### DISCUSSION TITLE
Comprehensive guide to JavaScript performance analysis using Chrome DevTools

### DISCUSSION THREAD

**Discussion by giscus** [jiayihu/blog#69] [Announcements]
# Comprehensive guide to JavaScript performance analysis using Chrome DevTools

Let's see how to navigate the Chrome Devtools Performance tab to effectively analyse and improve the performance of your JavaScript while avoiding common errors. Our use case will be improving the rendering FPS of a real-world canvas library.

https://blog.jiayihu.net/comprenhensive-guide-chrome-performance/

**Comment by johncerpa** [comment]
Excellent article. Great read

**Comment by liuliangsir** [comment]
Awesome

**Comment by AlexandrBukhtatyy** [comment]
I was loocking for steal but found gold
Thank You for the article

**Reply by AlexandrBukhtatyy** [reply]
can i translate this article on my native language? with link on your blog?

**Reply by jiayihu** [reply]
Thanks, sure go ahead 😄

**Comment by yairEO** [comment]
It's very difficult to learn from such an extremely long article with a few screenshots and detached code pieces 

if this was a YouTube video it would have been a thousand times easier to consume this content's wisdom

**Reply by jiayihu** [reply]
Thanks for the feedback. I'm planning to propose the content as Call for Paper for some conferences so hopefully there will be a video version as well.

Meanwhile, is there any particular section that's hard to follow?

**Comment by YoonHan** [comment]
Your post was extremely detailed and helpful. I look forward to your future posts :)
